features/cli/help.feature

```diff
@@ -182,6 +182,7 @@ Feature: Pro Client help text
         --format {cli,json}  output in the specified format (default: cli)
         --variant VARIANT    The name of the variant to use when enabling the
                              service
+        --auto               enable all default services
```

Comment:
The help currently seems to be kind-of sorted alphabetically (with two small exceptions).

Shouldn't we keep it that way, and actually improve making sure it's all sorted alphabetically apart from `-h`?